src/libm/dispavx.c.org

```diff
@@ -138,4 +138,28 @@ static int cpuSupportsFMA4() {
   static CONST fptype (*pfn)(fptype arg0, fptype arg1, fptype arg2) = dfn; \
   EXPORT CONST fptype funcName(fptype arg0, fptype arg1, fptype arg2) { return (*pfn)(arg0, arg1, arg2); }
 
+#define DISPATCH_si_si(funcName, pfn, dfn, funcavx, funcfma4, funcavx2) \
```

Comment:
Can you explain what DSPATCH_si_si and DISPATCH_sp_si names are for? In particular, what si and sp stand for?

Reply:
There is not much meaning. si is int, sp is void *.

Reply:
The names don't make it clear. I think it is worth adding a comment at the top of the sources saying what the DISPATCH_A_B macro mean, for each of the values of A and B.

Reply:
Actually I wanted to rename _d to _vd or _vf, and use _i for int. I will also do this.